### ISSUE TITLE
[kernel-lts2019] `testing the buffer` noise

### ISSUE THREAD

**Issue by KyleSanderson** [clearlinux/distribution#1878]
Patch here: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=41c55ea6c2a7ca4c663eeec05bdf54f4e2419699

Impacts 5.4.28+ (including 5.4.30).

**Comment by ahkok** [comment]
Is this merged into the LTS branch yet?

@miguelinux

**Comment by jeka** [comment]
That affects 5.5.15-440.kvm as well. 
I was playing with k3s and noise in the journal is unbearable. 
```
$ uptime
20:28:15  up   0:24,  1 user,  load average: 0.00, 0.04, 0.13

$ journalctl -b | grep "testing the buffer" | wc -l
11928
```
Also it makes journal to rotate really fast.

**Comment by miguelinux** [comment]
The fix is applied in linux-lts2019-5.4.31-27

**Comment by miguelinux** [comment]
> The fix is applied in linux-lts2019-5.4.31-27
> 

The Clear Linux release 32810 is out

**Comment by KyleSanderson** [comment]
ACK. Thanks team
